libraries/botbuilder-dialogs/package.json

```diff
@@ -26,7 +26,8 @@
     "@microsoft/recognizers-text-suite": "1.1.4",
     "@types/node": "^10.17.27",
     "botbuilder-core": "4.1.6",
-    "globalize": "^1.4.2"
+    "globalize": "^1.4.2",
+    "line-reader": "^0.4.0"
```

Comment:
This should be a dev dependency as it’s only used for testing.

Reply:
Hi @joshgummersall we just pushed the changes. Thanks!